gems/rack-attack/OSVDB-132234.yml

```diff
@@ -0,0 +1,27 @@
+---
+gem: rack-attack
+framework: rails
```

Comment:
Is my understanding of the `framework` field correct here? Using it to mean only when used with Rails are you affected by this issue...